src/services/teams/index.js

```diff
@@ -712,4 +712,34 @@ module.exports = function setup() {
 		before: hooks.beforeAdmin,
 		after: hooks.afterAdmin,
 	});
+
+
+	ScopePermissionService.initialize(app, '/teams/:scopeId/userPermissions', async (userId, team) => {
```

Comment:
Aktuell sind noch alle weiteren Team Services in diesem File. Es würde aber Sinn machen, diese auszulagern in teams/services/*. Daher wäre es gut, wenn schon mal die beiden Services dort landen.

Reply:
nice to have